source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -1,871 +1,865 @@
-files, and two bundle directories: :file:`local-bundles` and
-:file:`upstream-bundles`.
+This example shows the basic steps for first time setup of mixer for a new mix.
 
-If you wish to start with a different version of upstream |CL| or a
-different initial mix version, you can specify these options as flags.
-For example:
+#. Create an empty directory to use as a workspace for mixer:
 
-.. code-block:: bash
+   .. code-block:: bash
 
-   mixer init --upstream-version 21060 --mix-version 10
+      mkdir ~/mixer
 
+#. In your mixer workspace, generate an initial mix based on the latest upstream
+   |CL| version, with minimum bundles:
 
-Additionally, to build a mix with your own custom RPMs, use the optional
-*--local-rpms* flag, for example:
+   .. code-block:: bash
 
-.. code-block:: bash
+      cd ~/mixer
+      mixer init
 
-   mixer init --local-rpms
+   Note in the initialization output, that your initial mix version is set to
+   10 and that the minimum bundles have been added.
 
-This command creates the :file:`local-yum` and :file:`local-rpms`
-directories in your mix workspace and adds their paths to the generated
-:file:`builder.conf`. If the configuration file already exists, you must add
-the paths manually. For more information on using these directories or
-setting them up manually, see `Create or locate RPMs for the mix`_.
+#. Edit :file:`builder.conf` to set the value of CONTENTURL and VERSIONURL to
+   the IP address of your content server:
 
-If all upstream |CL| bundles will be part of the mix, you can easily add 
-them all during initialization with the optional *--all-upstream* flag. For example:
+   .. code-block:: bash
 
-.. code-block:: bash
+      nano builder.conf
 
-   mixer init --all-upstream
+   Use the IP address from the nginx server you set up in the prerequisite
+   `Set up a nginx web server for mixer`_. For example:
 
-Finally, you may want to track the contents of your mixer workspace with a
-git repository. This is a great way to track changes to your mix's content
-or to revert to earlier versions if something goes wrong. Mixer can set this
-up automatically with the optional *--git* flag, for example:
+   .. code-block:: console
 
-.. code-block:: bash
+      CONTENTURL="http://192.168.25.52"
+      VERSIONURL="http://192.168.25.52"
 
-   mixer init --git
 
-.. note::
-   You can use any or all of the above optional flags at the same time, for example:
 
-   .. code-block:: bash
+Example 2: Create a simple mix
+==============================
 
-      mixer init --upstream-version 21060 --mix-version 10 --local-rpms --all-upstream --git
+This example shows how to create a simple custom mix using upstream content.
+We'll create an image for a QEMU virtual machine which we can later use to test
+our mix.
 
-Edit builder.conf
-=================
+We can use the default bundles that were added during intialization, but these
+include the :command:`native-kernel` bundle which has hundreds of drivers we
```

Comment:
"which has hundreds of drivers we don't need for the QEMU virtual machine." --> "which is intended to be use on a bare metal system instead of a VM."